week1_intro/gym_interface.ipynb

```diff
@@ -75,13 +75,13 @@
    "source": [
     "### Gym interface\n",
     "\n",
-    "The three main methods of an environment are\n",
-    "* `reset()`: reset environment to the initial state, _return first observation_\n",
-    "* `render()`: show current environment state (a more colorful version :) )\n",
-    "* `step(a)`: commit action `a` and return `(new_observation, reward, is_done, info)`\n",
+    "The three main methods of this environment are:\n",
```

Comment:
It should be "an environment", since there are many possible environments, and the interface is identical for all of them.